02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/🗂️ CSS Snippets.md

```diff
@@ -9,16 +9,3 @@ publish: true
 # 🗂️ CSS Snippets
 
 #placeholder/description 
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
```

Comment:
I'd keep these as a manual MOC for now (once more snippets are added, we'll have to adjust).